src/main/common/sensor_alignment.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "axis.h"
+#include "maths.h"
+
+typedef enum {
+    ALIGN_DEFAULT = 0, // driver-provided alignment
+
+    CW0_DEG = 1,
+    CW90_DEG = 2,
+    CW180_DEG = 3,
+    CW270_DEG = 4,
+    CW0_DEG_FLIP = 5,
+    CW90_DEG_FLIP = 6,
+    CW180_DEG_FLIP = 7,
+    CW270_DEG_FLIP = 8,
+
+    ALIGN_CUSTOM = 9, // arbitrary sensor angles, e.g. for external sensors
+} sensor_align_e;
+
+typedef union sensorAlignment_u {
+    // value order is the same as axis_e
+
+    int16_t raw[XYZ_AXIS_COUNT];
+    struct {
+        int16_t roll;
+        int16_t pitch;
+        int16_t yaw;
+    } values;
```

Comment:
(this can easily be unnamed struct. `.values.` caries little in code)

Reply:
`accelerometerTrims`  has the same issue.

Reply:
done, not fixed `accelerometerTrims` which is out of scope of this PR.

Reply:
using anonymous inner structure causes a problem with assignments on the SITL build on the build server, check out this log:

https://travis-ci.com/betaflight/betaflight/jobs/213142356#L890

and note that .yaw *is* defined!

I don't have compilation problems here for the unit tests (cygwin gcc compiler) or firmware (arm gcc compiler)...

to solve this issue I just used memset in a later commit, see 9db938549cd24ba35ea70c2454fc6f5d4c586607